usaspending_api/agencies/management/commands/load_agencies.py

```diff
@@ -0,0 +1,270 @@
+            action="store_true",
+            help=(
+                "Reloads agencies even if the change max change threshold of {:,} is exceeded.  This is a safety "
+                "precaution to prevent accidentally updating every award, transaction, and subaward in the system as "
+                "part of the nightly pipeline.".format(MAX_CHANGES)
+            ),
+        )
+
+    def handle(self, *args, **options):
+
+        self.agency_file = options["agency_file"]
+        self.force = options["force"]
+
+        logger.info("AGENCY FILE: {}".format(self.agency_file))
+        logger.info("FORCE SWITCH: {}".format(self.force))
+        logger.info("MAX CHANGE LIMIT: {}".format("unlimited" if self.force else "{:,}".format(MAX_CHANGES)))
+
+        with Timer("Load agencies"):
+            try:
+                with transaction.atomic():
+                    self._perform_load()
+                    t = Timer("Commit agency transaction")
+                    t.log_starting_message()
+                t.log_success_message()
+            except Exception:
+                logger.error("ALL CHANGES ROLLED BACK DUE TO EXCEPTION")
+                raise
+
+            try:
+                self._vacuum_tables()
+            except Exception:
+                logger.error("CHANGES WERE SUCCESSFULLY COMMITTED EVEN THOUGH VACUUMS FAILED")
+                raise
+
+    def _read_raw_agencies_csv(self):
+        agencies = read_csv_file_as_list_of_dictionaries(self.agency_file)
+        if len(agencies) < 1:
+            raise RuntimeError("Agency file '{}' appears to be empty".format(self.agency_file))
+
+        self.agencies = [
+            Agency(
+                row_number=row_number + 1,
+                cgac_agency_code=prep(agency["CGAC AGENCY CODE"]),
+                agency_name=prep(agency["AGENCY NAME"]),
+                agency_abbreviation=prep(agency["AGENCY ABBREVIATION"]),
+                frec=prep(agency["FREC"]),
+                frec_entity_description=prep(agency["FREC Entity Description"]),
+                frec_abbreviation=prep(agency["FREC ABBREVIATION"]),
+                subtier_code=prep(agency["SUBTIER CODE"]),
+                subtier_name=prep(agency["SUBTIER NAME"]),
+                subtier_abbreviation=prep(agency["SUBTIER ABBREVIATION"]),
+                toptier_flag=bool(strtobool(prep(agency["TOPTIER_FLAG"]))),
+                is_frec=bool(strtobool(prep(agency["IS_FREC"]))),
+                user_selectable=bool(strtobool(prep(agency["USER SELECTABLE ON USASPENDING.GOV"]))),
+                mission=prep(agency["MISSION"]),
+                website=prep(agency["WEBSITE"]),
+                congressional_justification=prep(agency["CONGRESSIONAL JUSTIFICATION"]),
+                icon_filename=prep(agency["ICON FILENAME"]),
+                include_toptier_without_subtier=prep(agency["CGAC AGENCY CODE"]) in DOD_ARMED_FORCES_CGAC,
+            )
+            for row_number, agency in enumerate(agencies)
```

Comment:
Could set `start=1` in enumerate() 
https://docs.python.org/3/library/functions.html#enumerate